WebDriverAgentLib/Categories/XCUIApplication+FBHelpers.m

```diff
@@ -108,4 +108,19 @@ + (NSDictionary *)accessibilityInfoForElement:(XCElementSnapshot *)snapshot
   return info;
 }
 
+-(void) fb_resolve
+{
+  if (self.fb_isActivateSupported) {
+    
+    if(self.state != XCUIApplicationStateRunningForeground)
```

Comment:
XCUIApplicationStateRunningForeground won't work for Xcode < 9, since it's unknown there.

Reply:
Yes but the self.fb_isActivateSupported will check that.
 
activate and state where added to the api at the same time.

ie. If activate is supported then so is state